frontend/eslint.config.js

```diff
@@ -0,0 +1,69 @@
+import react from "eslint-plugin-react"
+import hooks from "eslint-plugin-react-hooks"
+import reactRefresh from "eslint-plugin-react-refresh"
+import love from "eslint-config-love"
+import tsParser from "@typescript-eslint/parser"
+import globals from "globals"
+
+export default [
+  love,
+  {
+    rules: {
+      'react-refresh/only-export-components': 'warn',
+    }
+  },
+  {
+    files: [
+      '**/*.ts',
+      '**/*.tsx'
+    ],
+    rules: {
+      '@typescript-eslint/space-before-function-paren': [
+        'error',
+        {
+          anonymous: 'always',
+          named: 'never',
+          asyncArrow: 'always'
+        }
+      ],
+      'no-undef': 'off'
+    }
+  },
+  {
+    ignores: [
+      'dist/**',
+      'eslint.config.js'
+    ]
+
+  },
+  {
+    linterOptions: {
+      reportUnusedDisableDirectives: true
+    }
+  },
+  {
+    settings: {
+      react: { version: 'detect' }
+    }
+  },
+  {
+    languageOptions: {
+      parser: tsParser,
+      parserOptions: {
+        ecmaVersion: 'latest'
+      },
+      globals: {
+        ...globals.browser,
+        ...globals.es2020,
+        ...globals.node,
+      }
+    }
```

Comment:
I'm curious: What exactly does this `global` impact / what happens if we don't include it?

Reply:
I don't notice any different behavior when it is not included. This was included as carryover from the original implementation, which was a boilerplate eslint config file. Here is the purpose that globals serve per the eslint docs [here](https://eslint.org/docs/latest/use/configure/language-options#specifying-globals) and [here](https://eslint.org/docs/latest/use/configure/language-options#predefined-global-variables). Looks like if you have global variables and want particular ESLint rules to apply to said variables, then you need to implement `globals`.

I've opted to remove them, but we have at least captured how to implement them in the new schema if something arises later down the line.